src/app.py

```diff
@@ -3016,7 +2992,7 @@ def get_prov_info_for_dataset(id):
     # Token is not required, but if an invalid token provided,
     # we need to tell the client with a 401 error
     validate_token_if_auth_header_exists(request)
-
+    organ_types_dict = schema_manager.get_organ_types()
```

Comment:
Found duplicate in `/datasets/<id>/prov-info` too

Reply:
I don't see the duplicates in the sankey data PR, could just be in this prov-info branch. 

That's good performance optimization you did even though that organ call internally is being cached.

Reply:
Yes my concern was less about the speed of the prov-info endpoints (since as you mentioned, its cached) but rather clogging up ontology with thousands of requests.